database/sql/schema_version_26.sql

```diff
@@ -0,0 +1,2 @@
+create extension if not exists pgcrypto;
```

Comment:
I noticed that using Postgresql extensions makes the software harder to install for people not familiar with Postgesql. That requires superuser privileges. Generating a random tokens can be done in Go. Miniflux provides some functions for that.

Reply:
Migration 2 is a bad example. The `hstore` column could have been a `jsonb` column to avoid superuser privileges.

Don't use a default value, just set it to null. Generate a token when a user decide to share an article.